fusor-ember-cli/app/components/osp-node.js

```diff
@@ -0,0 +1,85 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  classNames: ['row osp-node-row'],
+
+  deleteEnabled: true,
+
+  label: Ember.computed('node', 'ports', function() {
+    let node = this.get('node');
+    let macAddress = node ? node.getMacAddress(this.get('ports')): null;
+    return macAddress || node.get('id');
+  }),
+
+  safeLabel: Ember.computed('label', function() {
+    let label = this.get('label');
+    return label ? label.replace(/[^A-Z0-9]/ig, '') : '';
+  }),
+
+  status: Ember.computed('node', function() {
+    if (this.get('node.last_error')) {
+      return 'Error';
+    }
+
+    if (Ember.isPresent(this.get('node.provision_state'))) {
+      return 'node.provision_state';
+    }
+
+    return 'Free';
+  }),
+
+  introspectionTask: Ember.computed('node', 'introspectionTasks.@each', function() {
+    return this.get('node').getIntrospectionTask(this.get('introspectionTasks'));
+  }),
+
+  foremanTask: Ember.computed('introspectionTasks.@each', 'foremanTasks.@each', function() {
+    return this.get('node').getForemanTask(this.get('introspectionTasks'), this.get('foremanTasks'));
+  }),
+
+  isNodeReady: Ember.computed('node.properties.cpu', 'node.properties.memory_mb', 'node.properties.local_gb', function() {
+    return this.get('node.ready');
+  }),
+
+  isNodeInspecting: Ember.computed('node.ready', 'foremanTask', 'foremanTask.state', 'foremanTask.result', function() {
+    return !this.get('node.ready') &&
+      this.get('foremanTask') &&
+      this.get('foremanTask.state') === 'running' &&
+      this.get('foremanTask.result') === 'pending';
+  }),
+
+  isNodeError: Ember.computed(
+    'isNodeReady',
+    'isNodeInspecting',
+    'foremanTask',
+    'foremanTask.state',
```

Comment:
Computed result doesn't depend on `foremanTask.state`. As a rule of thumb I don't include anything in the list of dependencies that the value doesn't directly depend on. If it actually *does* depend on this prop, it's an indication the calc logic should use it in order to be semantically correct.